### DISCUSSION TITLE
How To Disable Caching On a Page

### DISCUSSION THREAD

**Discussion by goddrummerofdw** [vercel/community#3276] [Help]
### Summary

Is there a way to disable caching on a page? (not a fetch call)

### Steps to Reproduce

Nothing to preview ](https://github.com/orgs/vercel/discussions/3259

**Comment by amyegan** [comment]
Hey @goddrummerofdw. I don't know exactly what you're trying to accomplish, so I don't have a specific recommendation. But you can find more information on cache configuration in the documentation.
- [Caching on Vercel's Edge Network](https://vercel.com/docs/concepts/edge-network/caching#)
- [Vercel Data Cache](https://vercel.com/docs/infrastructure/data-cache#)
- [Caching Serverless Function Responses](https://vercel.com/docs/concepts/functions/serverless-functions/edge-caching#)
- [Static Files Caching](https://vercel.com/docs/concepts/edge-network/caching#static-files-caching)
- [How to Configure the Cache-Control Response Header in Vercel Projects](https://vercel.com/guides/how-to-configure-the-cache-control-response-header-in-vercel-projects)

**Reply by goddrummerofdw** [reply]
Hi @amyegan , thanks for getting back.  I implemented some of what the documentation you provided suggested but still running into the same issue 

I should have been more detailed but I left a link to a different ticket I created that has a little more info  on why I’d like to disable page caching. 

https://github.com/orgs/vercel/discussions/3259
